ansible/group_vars/sonic/vars

```diff
@@ -1,9 +1,9 @@
 ansible_ssh_user: admin
-ansible_ssh_pass: password
-ansible_become_pass: password
+ansible_ssh_pass: admin
+ansible_become_pass: admin
```

Comment:
Thanks for catching this. I don't like the default password in sonic-buildimages. Could you also change it to "YourPaSsWoRd" at both places?